base/src/main/java/io/spine/validate/constraint/ConstraintOf.java

```diff
@@ -28,13 +28,13 @@
 import java.util.Optional;
 
 /**
- * An external validation for a field.
+ * An external constraint for a field.
  *
- * <p>Contains information about validation of another field, described by the option value.
+ * <p>Contains information about constraint of another field, described by the option value.
  */
-final class ValidationOf extends MessageOption<String> {
+final class ConstraintOf extends MessageOption<String> {
```

Comment:
This option needs to be called `ValidationOf` — as we are speaking of the process. And in Protobuf it's still named `validation_of`.